test/ci-scripts/Write-TestResults.ps1

```diff
@@ -508,10 +499,6 @@ $badges = @(
     @{
         "url"      = "https://img.shields.io/badge/Bicep%20Version-$BicepVersion-/?color=$BicepVersionColor";
         "filename" = "BicepVersion.svg"
-    },
-    @{
-        "url"      = "https://img.shields.io/badge/Template%20Analyzer%20Check-$TemplateAnalyzerReportedErrors-/?color=$TemplateAnalyzerResultColor";
-        "filename" = "TemplateAnalyzerResultColor.svg"
```

Comment:
change the filename to just ```TemplateAnalyzerResult.svg```

Reply:
Kind of completely unrelated, but, while making the change I noticed this:
```
switch ($BestPracticeResult) {
    "PASS" { $BestPracticeResultColor = "brightgreen" }
    "FAIL" { $BestPracticeResultColor = "red" }
    default {
        $BestPracticeResult = $na
        $BestPracticeResult = "inactive"  
    }
}
```
It's just overwriting the same variable instead of using a $...ResultColor variable like the rest, is this an issue that should be reported?

Reply:
it is wrong - though there aren't really any designed cases where that would hit the default... still if you don't mind fixing that would be good.